elastic/kibana
# [Infrastructure UI] Implemented Telemetry for Hosts View Logs Tab

closes [#974](https://github.com/elastic/obs-infraobs-team/issues/974)

## Summary
This PR adds telemetry tracking for queries submitted in the logs tab.

## Testing

1. Navigate to the Hosts View and click on the Logs Tab
2. Type any search query in the search bar
3. Wait for some minutes then go to [FS](https://app.fullstory.com/ui/1397FY/dashboards) 
4. you will see the new queries under the `Hosts View Logs Query Testing ` Dashboard

<img width="1087" alt="Screenshot 2023-04-03 at 18 39 16" src="https://user-images.githubusercontent.com/11225826/229585725-c1922b11-a104-4242-bf9d-bc857e470e9d.png">
